sql/mob_skill_lists.sql

```diff
@@ -3572,7 +3572,10 @@ INSERT INTO `mob_skill_lists` VALUES ('TRUST_Uka_Totlihn',1062,167); -- Judgemen
 -- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Flaviria_UC',1072,0);
 -- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Babban',1073,0);
 -- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Abenzio',1074,0);
--- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Rughadjeen',1075,0);
+-- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Rughadjeen',1075,0); -- Power Slash
+-- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Rughadjeen',1075,0); -- Sickle Moon
+-- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Rughadjeen',1075,0); -- Ground Strike
+-- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Rughadjeen',1075,0); -- Victory Beacon
```

Comment:
Are you going to add these in?

Reply:
> Will need Retail captures of Rughadjeen's WS to add the correct IDs (action and animation), left WS commented out for now.

In the future, that is my hope. Just need retail data to do so. Left them commented out to prevent weird animation behavior for now.